panther_analysis_tool/main.py

```diff
@@ -810,6 +828,61 @@ def setup_parser() -> argparse.ArgumentParser:
     return parser
 
 
+def zip_managed_schemas(args: argparse.Namespace) -> Tuple[int, str]:
```

Comment:
Is the proposed process to: 1. create a release in github 2. after the release is created, run `pat` to generate the assets? 3. update the release to include the assets?

Reply:
Yes, the `vX.Y.Z` tag must exist to produce the asset.

Reply:
If we ever automate this we just need to do the `git tag` before we issue a 'create release' git hub api call.